packages/url/src/prepend-https.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+import { prependHTTP } from './prepend-http';
+
+/**
+ * Prepends "https://" to a url, if it looks like something that is meant to be a TLD.
+ *
+ * @param {string} url The URL to test.
+ *
+ * @example
+ * ```js
+ * const actualURL = prependHTTPS( 'wordpress.org' ); // https://wordpress.org
+ * ```
+ *
+ * @return {string} The updated URL.
+ */
+export function prependHTTPS( url ) {
+	if ( ! url ) {
+		return url;
+	}
+
+	// If url starts with http://, return it as is.
+	if ( url.startsWith( 'http://' ) ) {
+		return url;
+	}
```

Comment:
I guess this ought to check for `https` as well.

Reply:
Alternatively, it does make sense to keep this function just as "prepend if there is no protocol", because it's named "prepend HTTPS" and not "change protocol". If that matches your intention, then we'll just need to alter `url.startsWith( 'http://' )` to `url.startsWith( 'http://' ) || url.startsWith( 'https://' )`

Reply:
I felt that it already had a protocol we should ignore it. So your suggestion makes sense.